### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by segg21**
Don't merge until complete

Todo
- [x] Leaderboard
- [x] Names/mass
- [ ] Chat

Todo Last
- [ ] Spectating
- [ ] Animations
- [ ] Socket

### CODE REVIEW

**Comment by segg21** [comment]
@Andrews54757 I've done an example viewbox, with the container as a camera in my trials of client dev. Alot of the code I gave you should pretty much build the basics of the canvas, in which I have already done. It's just the little tiny bugs i've came across, in which needs to be re-worked. Kindof why I was testing with other engines.

As of re-thinking, i've thought of another way of doing things. As of now, it's almost 8am, I have to do some things this morning. I'll back before noon.

File: openAgarClient.js [hunk 1]
```diff
@@ -5,251 +5,415 @@
...
+    stage = new PIXI.Container();
+
+    // Create camera
+    camera = new PIXI.Container();
+
+	
+    // Create Chat
+    chat.container = new PIXI.Container()
+    let chat_graphics = new PIXI.Graphics();
+    chat_graphics.alpha = .9
```
Comment (Andrews54757) [inline openAgarClient.js]: chat_graphics, chat_placeholder, and leaderboard_graphics should be accessible outside the function. @LegitSoulja

Comment (segg21) [reply openAgarClient.js]: @Andrews54757 it will have to be re-drawn. Not stored.

Comment (segg21) [reply openAgarClient.js]: chat_placeholder shouldn't be touched, it's supposed to be an input. I just put text there for an example.

Comment (segg21) [reply openAgarClient.js]: @Andrews54757 Ok, I see the reasons of needing it stored, in your scenario with the animations.

I was thinking, any changes we make to the cell, we draw a new graphics/texture and apply it to the sprite itself, but i don't know, and cant determine the way you're doing it.

So the option of changing the graphics (dynamically) is there for your needs.

**Comment by Andrews54757** [comment]
Lets merge this for now, to resolve conflicts

**Pull request merged**